addons/mail/static/src/models/channel_member_list_category_view.js

```diff
@@ -8,6 +8,8 @@ import { sprintf } from '@web/core/utils/strings';
 
 registerModel({
     name: 'ChannelMemberListCategoryView',
+    template: 'mail.ChannelMemberListCategoryView',
+    templateGetter: 'record',
```

Comment:
`record` could be the default value of templateGetter, so that we can sometimes avoid repeating it in the model definition

Reply:
Could be, but I didn't do it yet just because all of them define a `getter`, and it's best to keep diff as small as possible for such a change in this PR.

Reply:
Wouldn't that reduce the diff?

Reply:
About the same actually. Just be aware there's implicit `record` getter if not provided.
I did that, so not necessarily needed to provide `templateGetter` and we can just use `record` getter then (there are 2 templates that use it right now)